prow/cluster/starter.yaml

```diff
@@ -14,7 +14,7 @@ kind: ConfigMap
 metadata:
   name: config
 data:
-  config: |
+  config.yaml: |
```

Comment:
Do we need to make the same change for the `plugins` key of the `plugins` configmap?